cl/opinion_page/templates/includes/rd_download_button.html

```diff
@@ -27,17 +27,18 @@
       {% endif %}
       {% if rd.pacer_url %}
         <li>
-          {% if not request.COOKIES.buy_on_pacer_modal and not request.COOKIES.recap_install_plea %}
             <a href="{{ rd.pacer_url }}"
-            data-toggle="modal" data-target="#modal-buy-pacer"
-            class="open_buy_pacer_modal"
-            target="_blank"
-            rel="nofollow">Buy on PACER</a>
-          {% else%}
-            <a href="{{ rd.pacer_url }}"
-            target="_blank"
-            rel="nofollow">Buy on PACER</a>
-          {% endif %}
+              {% if not request.COOKIES.buy_on_pacer_modal and not request.COOKIES.recap_install_plea %}
+                data-toggle="modal" data-target="#modal-buy-pacer"
+                class="open_buy_pacer_modal"
+             {% endif %}
+              target="_blank"
+              rel="nofollow">Buy on PACER</a>
+            {% if rd.pacerdash_url %}
+              <a href="{{ rd.pacerdash_url }}"
+              target="_blank"
+              rel="nofollow">Buy on PacerDash</a>
+            {% endif %}
```

Comment:
This comment is more for @mlissner but wouldnt this all be nicer if we built the html link at the filter level with something like 

```
@register.filter
def pacerdash_url(rd: RECAPDocument) -> str:
    """"""
    if not rd.pacerdash_url:
        return ""
    return mark_safe(
        f'<a href="{ rd.pacerdash_url }" target="_blank" rel="nofollow">Buy on PacerDash</a>'
    )
```

and then we would just pass 

`{% pacerdash_url %}`

into the html template and really simplify the HTML